src/analysis/functions.rs

```diff
@@ -567,7 +567,7 @@ fn analyze_function(
                     || name.starts_with("new_with")
```

Comment:
Fyi there's another `filter_map().next()` just 10 lines above this.

Reply:
Couldn't find it.

Reply:
```rust
    let mut new_name = configured_functions
        .iter()
        .filter_map(|f| f.rename.clone())
        .next()
```

At line 554-557.

Reply:
Thanks. Didn't look in the right file...
I think I had more of these in https://github.com/gtk-rs/gir/pull/1092, but that can wait.

Reply:
Thanks! I agree, better make a separate PR that enables some more strict clippy lints and fix them there :)